core/common/src/test/java/alluxio/AlluxioURITest.java

```diff
@@ -37,7 +39,7 @@
   @Test
   public void basicAlluxioUri() {
     AlluxioURI uri = new AlluxioURI("alluxio://localhost:19998/xy z/a b c");
-    assertEquals("localhost:19998", uri.getAuthority());
+    assertEquals("localhost:19998", uri.getAuthority().toString());
```

Comment:
also add assert on `AuthorityType` since you added the getter? same for the following tests

Reply:
Added, thanks